content/nic/technical-specifications.md

```diff
@@ -17,17 +17,16 @@ We recommend using the latest release of NGINX Ingress Controller. We provide so
 We test NGINX Ingress Controller on a range of Kubernetes platforms for each release, and list them in the [release notes]({{< ref "/nic/releases.md" >}}). We provide technical support for NGINX Ingress Controller on any Kubernetes platform that is currently supported by its provider, and that passes the [Kubernetes conformance tests](https://www.cncf.io/certification/software-conformance/).
```

Comment:
Might be helpful for users to mention the EoL policy mentioned in an internal discussion.

Reply:
I had a slight brainfart. I think there are actually improvements to be made to the phrasing, but I will do them myself at a later point when revisiting this page. My bad!